src/apis/WikipediaApi.py

```diff
@@ -57,10 +58,10 @@ def get_random_text_and_links(self):
     def extract_text_and_links(page):
         response = BeautifulSoup(page.text, 'html.parser')
         links = response.find_all('a')
-        # TODO: This link parsing needs to be fixed to get all the links we want and nothing else
-        unique_links = set([link["href"].split("/")[-1].split("#")[0]
+        prefix = "./"
```

Comment:
Shouldn't the prefix be './wiki/'

Unless I'm not understanding the code?

Reply:
The html returned from the API is different from just viewing the html on the pages themselves; internal links are all of the format `./banana`.